issue-106552.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
```

Comment:
This file should not be in the root.
You should delete, it is also in the tests/ directory.

Reply:
Whoops, don't know how that got there :confused: